src/app/home/partners/partners.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+
+import { Observable } from 'rxjs';
+
+import { PartnerService } from 'src/app/core/services/partner.service';
+import { Partner } from 'src/app/core/model/partner';
```

Comment:
To be consistent across the app, try to avoid importing from src. Use relative paths.